tests/test_vectors/discrete_gauss_100.json

```diff
@@ -0,0 +1,56 @@
+{
```

Comment:
In my opinion it's too early to have test vectors. If you'd like to test that the implementation hasn't changed (this is definitely useful), then I'd suggest folding this into unit tests in the `dp::distributions` module.

If @divergentdave or someone else asked for test vectors, then feel free to disregard this comment.

Reply:
it's how i understood [this converation](https://github.com/divviup/libprio-rs/pull/578#discussion_r1263128510) but i may have misread. should i maybe rename/move the folder until the specification work has progressed?

Reply:
Thanks for the reminder ... no I think just leave as is.